src/modules/crud.js

```diff
@@ -0,0 +1,70 @@
+import Storage from './storage.js';
+import { taskList, renderList } from './render.js';
+import Task from './task.js';
+
+export default class List {
+  constructor() {
+    this.myTasks = [];
+    this.store = new Storage();
```

Comment:
The site works very well with a perfect description and readme file.
_But there are some little things which can make it better._

One of the few things which need some checking is, the declaration of an object inside a constructor.